packages/runner-ct/src/lib/ResizableBox.tsx

```diff
@@ -66,6 +66,7 @@ export const ResizableBox: React.FC<ResizableBoxProps> = ({
       {!disabled && (
         <div
           data-cy="resizer"
+          className="Resizer"
```

Comment:
Since we are doing the same thing as the resize vertical, why not keep the style intact?

```suggestion
          className="Resizer vertical"
```